modules/macros/pages/link-macro-attribute-parsing.adoc

```diff
@@ -1,47 +1,45 @@
 = Link Macro Attribute Parsing
 
 If named attributes are detected between the square brackets of a link macro (including URL macros), that text is parsed as an attribute list.
-This page explains the conditions when this occurs and how to write the linked text so it does not get caught up in this parsing.
+This page explains the conditions when this occurs and how to write the link text so it does not get caught up in this parsing.
 
 == Linked text alongside named attributes
 
-Normally, the whole text between the square brackets of a link macro is treated as the linked text.
+Normally, the whole text between the square brackets of a link macro is treated as the link text.
 
 [source]
 ----
 https://discuss.asciidoctor.org[Discuss Asciidoctor]
 ----
 
-However, if the text contains both an equals sign (`=`) and a comma (`,`), and the comma comes after the equals sign, the text is parsed as an attribute list.
-In this case, the linked text is the first positional attribute.
+However, if the text contains an unescaped equals sign (`=`), the text is parsed as a comma-separated attribute list.
```

Comment:
The introduction of the "unescaped" modifier here can be easily misunderstood. I think it would be better to state the behavior when the text is not enclosed in double quotes. Then follow up in the next sentence to say that attribute parsing can be disabled for the first positional attribute by enclosing it in quotes.